.github/workflows/integration_test.yaml

```diff
@@ -7,24 +7,25 @@ jobs:
   integration-test-lxd:
     strategy:
       matrix:
-        series: [xenial, bionic, focal]
+        # ubuntu 18.04 is no longer supported and hence xenial cannot be tested.
```

Comment:
I think you can remove this comment. It feels a little odd to reference something that is not there anymore

Reply:
How about improving the comment to `# ubuntu 18.04 is no longer supported and hence xenial cannot be tested in Github Actions.`? Since xenial tests still exist in the testing suite and I think it provides value in testing the environments in which the charm is deployed to, at least when run manually. The `metadata.yaml` includes xenial as well in the series section.
```
series:
  - xenial
  - bionic
  - focal
```
I'm not quite sure about the environment that currently uses xenial (perhaps in prod?) but just in case that is the case, the option to test xenial should still exist in my opinion.
How does that sound to you?

Reply:
I think it is reasonable to put a note since the charm still supports xenial on paper we just don't have a way of testing it on xenial anymore. There are still some deployments on xenial out there for context

Reply:
I think mentioning that it can't be tested on **Github Actions** would be necessary information to have, since it can still be tested locally. 
```
# ubuntu 18.04 is no longer supported on Github Action runner instance types
# and hence ubuntu versions older than bionic cannot be tested on Github Actions.
```
Would this be better now? :)
Thanks for the comment